helm_deploy/hmpps-restricted-patients-api/values.yaml

```diff
@@ -58,9 +58,9 @@ generic-service:
       DATABASE_NAME: "database_name"
       DATABASE_USERNAME: "database_username"
       DATABASE_PASSWORD: "database_password"
-    restricted-patients-queue-for-offender-events:
+    prison-events-restricted-patients-queue:
```

Comment:
had to have a different secret since both queues will exist at the same time for a while